resource_management_mobile/src/app/auth/login/login.page.scss

```diff
@@ -0,0 +1,83 @@
+
+$font-weight:600;
```

Comment:
Create new file called theme.scss and add the variables on it

Reply:
Created a new file and variables were moved into the theme.scss file.